README.md

```diff
@@ -3,8 +3,16 @@
 ```
 cmsrel CMSSW_10_6_29
 cd CMSSW_10_6_29/src
-cmsenv
+# DO NOT run cmsenv
```

Comment:
hi @AlbertHsuNTUphys but for normal work flow we needed ``cmsenv`` right? 

or should we move the 
```
source script/env.sh
```
to here for root env?

Reply:
Yes, but we need root so should we move 
source script/env.sh
to have a unique root version while we run independently. 

Agree @AlbertHsuNTUphys ?